AjobK/Athena
# 349 settings

closes #349

<br />

 **What is it supposed to do**  
In this pull request, I created a settings component which we can use to edit and change

<br />

 **How can we test your branch**  
- You can test the branch by going to the profile page and deactivating the account.
- If you refresh, your profile shouldn't be visible
- Try login in, you should get a popup that the account is deactivated, and it should ask if you want to reactivate the account

<br />

 **Additional notes**  
Please be kind :)

<br />

**1. General checklist**

<br />

**2. If relevant, front-end checklist**

<br />

**3. If relevant, back-end checklist**

<br />

**4. If relevant, test these browsers**

---

**PR Rules**  
*1. Add relevant labels.*  
*2. Select at least two reviewers.*  
*3. Assign all reviewers allowed to merge the branch, including yourself.*  
*4. If relevant to the sprint, put related issue in 'Awaiting Approval' column on project board.*  
*5. Add to the corresponding [milestone](https://github.com/AjobK/Seaqull/milestones).*

<br />

**Reviewer info**  
*1. You can use aliases to refer to specific PR checklist items. For example 'Hey @JohnDoe, for this line please check 3.a and 1.d.'*  
*2. Where possible, please use the suggestions feature in github so the author has a better idea of the solution you had in mind.*   

<br />

> This form should be filled in by the author of this PR.
